mmdeploy/backend/sdk/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import importlib
+
+
+def is_available():
```

Comment:
sdk is not available since mmdeploy is built in `build/lib`. this import attempt will always be failed unless cwd=`build/lib`. 
Should we copy `mmdeploy_python` here and `importlib.util.find_spec('mmdeploy.backend.ncnn.mmdeploy_python') is not None`?

Reply:
```
python tools/test.py configs/mmdet/detection/detection_sdk_dynamic.py \
../mmdetection/configs/retinanet/retinanet_r50_fpn_1x_coco.py \
--model ./work-dirs/mmdet/retinanet/trt/end2end.engine \
--out ./work-dirs/retinanet/trt/eval.pkl \
--metrics bbox \
--device cuda

loading annotations into memory...
Done (t=0.42s)
creating index...
index created!
Traceback (most recent call last):
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/tools/test.py", line 139, in <module>
    main()
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/tools/test.py", line 106, in main
    model = task_processor.init_backend_model(args.model)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection.py", line 74, in init_backend_model
    model_files, self.model_cfg, self.deploy_cfg, device=self.device)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 718, in build_object_detection_model
    **kwargs)
  File "/home/PJLAB/maningsheng/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mmcv/utils/registry.py", line 212, in build
    return self.build_func(*args, **kwargs, registry=self)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 34, in __build_backend_model
    **kwargs)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 598, in __init__
    super().__init__(*args, **kwargs)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 59, in __init__
    backend=backend, backend_files=backend_files, device=device)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 77, in _init_wrapper
    deploy_cfg=self.deploy_cfg)
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/codebase/base/backend_model.py", line 82, in _build_wrapper
    from mmdeploy.backend.sdk import SDKWrapper
ImportError: cannot import name 'SDKWrapper' from 'mmdeploy.backend.sdk' (/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/backend/sdk/__init__.py)
```

Reply:
how about adding `build/lib` to `PYTHONPATH` & `LD_LIBRARY_PATH`?

Reply:
@lzhangzz Ok if we update the document.